### DISCUSSION TITLE
Organisation account for only 1 month?

### DISCUSSION THREAD

**Discussion by olbapablo** [opengisch/QField#3941] [Q&A]
I have been collecting data using QField Cloud last year in an academia/NGO collaborative project. We want to keep using QField Cloud, under an Organisation account as we need to have several data collectors working on a private project. 

We will not be using QField Cloud all year round and only need it for 1 month (March 2023) It is not clear to me from the billing documentation and FAQ if a subscription to an Organisation account can be done for one month only (possibly 2 onths, starting this week). If I select the Organisation plan today or in March do I have to be subsribed for the whole year or can I stop it at the end of the data collection?

**Comment by ZsanettMed** [comment]
Hi, 

Yes, you can have an organization account only for one or two month, you just have to cancel the subscription. Otherwise the trial org could be something for you: https://app.qfield.cloud/organizations/create/trial_organization/
